internal/command/postgres/barman.go

```diff
@@ -188,7 +188,10 @@ func runBarmanCreate(ctx context.Context) error {
 	}
 
 	// Restart policy
-	machineConfig.Restart = &fly.MachineRestart{Policy: fly.MachineRestartPolicyAlways}
+	machineConfig.Restart.Policy = fly.MachineRestartPolicyAlways
```

Comment:
this line doesn't look good. It fails if `machine.Config.Restart` is `nil` but also it is no-op considering the next line assignment.